readme.md

```diff
@@ -0,0 +1,57 @@
+
+# DockerUI
+
+## Description
+
+DockerUI is a web application for managing Docker containers. With it, you can create a new container based on selected, existing images and delete or reset the existing one. The application shows when the process was started, when it was finished and how long it was lasted. In the main view of the application, we have access to basic information about all containers, while on the appropriate subpages we have access to a view with detailed information about the selected container.
+The visual side of the application was created based on bootstrap-grid.
+
```

Comment:
I miss the `How to use` section. With informations how to run it in browser and mount docker socket. Moreover, the information about the fact that this app makes calls to systems existing docker.socket is very important to mention somewhere. I suggest to add this to first sentence of description `...for managing Docker containers by communicating with...`.